erpnext/stock/get_item_details.py

```diff
@@ -571,6 +572,9 @@ def get_item_tax_template(args, item, out):
 			item_tax_template = _get_item_tax_template(args, item_group_doc.taxes, out)
 			item_group = item_group_doc.parent_item_group
 
+	if args.child_doctype and item_tax_template:
```

Comment:
@vorasmit args is a normal dictionary, so this produces an `AttributeError: 'dict' object has no attribute 'child_doctype'`. You can use dot notation only on `frappe._dict` objects. For normal dictionaries, you'll have to use `args.get("child_doctype")`.